tests/test_annotator.py

```diff
@@ -18,6 +18,16 @@ class TestAnnotator(unittest.TestCase):
     def setUp(self):
         self.data = [[1, 2, 3], [2, 5, 7]]
         self.data2 = pd.DataFrame([[1, 2], [2, 5], [3, 7]], columns=["X", "Y"])
+        self.data3: dict[str, list[str]] = {
```

Comment:
I don't think we've used inline type hints, and I am not sure they add something in this case as IDEs can understand the type from static analysis of what follows, right? Same applies for line 60 and 66.

Reply:
I'm just used to doing it to be fair. But it is indeed not necessary, although I don't think IDEs can detect an error if for example, I want to add a str to an int without converting one to another, if I haven't type hinted them.

Reply:
You're right about the help it can provide, thanks for pointing it out! As we both agree it is not necessary here (especially in the testing environment), would you like to adapt the code following the review or would you prefer me to take over (adding a commit to yours before merging)?

Reply:
Hey, I plan on making the modifications it by the end of the week!

Reply:
Changes were committed.

Reply:
Hi @trevismd, is there anything else I should do?